test/github/test_github_api.sh

```
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+CWD="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)" && cd "$CWD"
+set -ex
+
+export SERVER_URL="http://localhost:8000"
+
+function finish {
+  exit_code=$?
```

Shellcheck

[warning] exit_code appears unused. Verify use (or export if used externally).

(SC2034)